beaglebone-ai64.coffee

```diff
@@ -0,0 +1,52 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BBB_FLASH = 'Insert the SD card while powered down. Power up the <%= TYPE_NAME %> while holding down the BOOT/SW1 button at the opposite side of the USB Type-C port.
+You need to keep it pressed until the LEDs start flashing wildly.'
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	instructions.BOARD_REPOWER
+]
+
+module.exports =
+	version: 1
+	slug: 'beaglebone-ai64'
+	aliases: [ 'beaglebone-ai-64' ]
+	name: 'BeagleBone AI-64'
+	arch: 'aarch64'
+	state: 'unreleased'
```

Comment:
Remove the `state`, it's not currently used and its confusing. If the coffee file is here the DT is supported, otherwise it's discontinued from the operating system perspective.